janusgraph-core/src/main/java/org/janusgraph/graphdb/tinkerpop/gremlin/server/auth/HMACAuthenticator.java

```diff
@@ -0,0 +1,225 @@
+// Copyright 2017 JanusGraph Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+package org.janusgraph.graphdb.tinkerpop.gremlin.server.auth;
```

Comment:
Please add a blank line below the license, before the package name — some IDEs hide the top-level boilerplate comments, and lacking a blank line here may cause them to hide this package line as well.